website/src/data/members.js

```diff
@@ -236,6 +237,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/zionmateo/',
         imageURL: zion,
       },
+      {

```

Comment:
It's not a huge deal, but if you could move your entry a little earlier so that the devs are all in alphabetic order by first name (a little weird, I know), I'd appreciate it!